src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/RunConcurrentRepresentationTests.kt

```diff
@@ -437,7 +439,10 @@ class CoroutinesRunConcurrentRepresentationTest : BaseRunConcurrentRepresentatio
     @Before
     fun setUp() {
         assumeFalse(isInTraceDebuggerMode) // unstable hash-code
-        assumeFalse(isJdk8) // TODO: investigate why test is unstable on JDK8
+        // TODO: investigate why test is unstable on these JDKs
```

Comment:
Note: do not forget to create an issue if we decide not to fix this in this PR

Reply:
There is an issue for this problem: https://github.com/JetBrains/lincheck/issues/593

Reply:
https://github.com/JetBrains/lincheck/issues/628